src/bpmncwpverify/bpmn.py

```diff
@@ -98,6 +124,12 @@ def accept(self, visitor: "BpmnVisitor") -> None:
         self.traverse_outflows_if_result(visitor, result)
         visitor.end_visit_task(self)
 
+    def cleanup(self) -> None:
```

Comment:
Here is another "cleanup" on names. I'm wondering if we need a better way to create "names" for Promela from the various `bpmn` elements. I have been thinking about this issue for sometime, but I don't have a good solution yet. 

We need a mapping from `bpmn` element id to valid Promela IDs, and that mapping needs to go both ways. We also need a way to map counter-examples in Promela back to `bpmn` element IDs and then map those IDs to something that is human readable.

Does this problem description make sense?

Reply:
Okay, yes this does make sense. So, if I understand correctly, we need a better way of naming our BPMN elements for the promela code that is generated. Then, this mapping from `bpmn_element.name -> promela_element.name` needs to be able to go from `promela.name -> bpmn_element.name` in order to convert promela back into bpmn elements which can then be converted to something human readable (through a visitor I am assuming). Is that correct? Also, what do you mean by counter example? Thanks!

Reply:
Also, is this something that should be done in a new branch?